spi/src/main/java/org/jboss/pnc/spi/builddriver/BuildDriverStatus.java

```diff
@@ -25,7 +25,21 @@
  * Created by <a href="mailto:matejonnet@gmail.com">Matej Lazar</a> on 2014-11-23.
  */
 public enum BuildDriverStatus {
-    SUCCESS, FAILED, UNSTABLE, BUILDING, ABORTED, CANCELLED, UNKNOWN;
+    SUCCESS(true), FAILED, UNSTABLE(true), BUILDING, ABORTED, CANCELLED, UNKNOWN;
```

Comment:
I would treat UNSTABLE as unsuccessful. It's true that build results can be consumed by others, but if you requested to run tests they should pass.